addOns/scripts/src/test/java/org/zaproxy/zap/extension/scripts/automation/ScriptJobUnitTest.java

```diff
@@ -676,6 +682,16 @@ void shouldFailToAddIfFileNotReadable() throws IOException {
         assertThat(progress.getErrors(), contains("!scripts.automation.error.file.cannotRead!"));
     }
 
+    private static void setNotReadable(Path file) throws IOException {
+        assumeTrue(
+                Files.getFileStore(file).supportsFileAttributeView(PosixFileAttributeView.class),
+                "Test requires support for POSIX file attributes.");
+        Set<PosixFilePermission> perms =
+                Files.readAttributes(file, PosixFileAttributes.class).permissions();
+        perms.remove(PosixFilePermission.OWNER_READ);
```

Comment:
Do we need to remove `GROUP_READ` and `OTHERS_READ` as well?

Reply:
It's enough to remove `OWNER_READ` for the test.